### ISSUE TITLE
PT007 raises a false positive for single-item strings

### ISSUE THREAD

**Issue by charliermarsh** [charliermarsh/ruff#2008]
Not sure if this is inherited from the flake8 plugin, but this is a false positive for PT007:

```python
@pytest.mark.parametrize("archs", [["x86_64"], ["arm64", "universal2"]])
```

If the item on the left is a string with no commas, then the thing on the right doesn't have to be a list of tuples, it can be a list of anything, including lists.

_Originally posted by @henryiii in https://github.com/charliermarsh/ruff/issues/1506#issuecomment-1397813925_
